snitchpy/__init__.py

```diff
@@ -731,6 +780,10 @@ def _resume_pipeline(self, cmd: protos.Command) -> bool:
 
         return True
 
+    def _handle_kv(self, cmd: protos.Command) -> bool:
```

Comment:
I think we should have a "Feature" table in the README's that is the same across all SDK's and they each list if they support the feature or not. Features like "peek [Check] rule: kv [X] http request: [X]" and so on.

Then we can just copy pasta and update it accordingly for each sdk?

Reply:
Added feature tracking table to internal SDK doc